### ISSUE TITLE
Annotation ClonflictAction.IGNORE doesn't work.

### ISSUE THREAD

**Issue by Hannibalem** [Raizlabs/DBFlow#711]
I have the following table:

`@Table(database = PrototypeDatabase.class)
public class Tag extends BaseModel {

    @PrimaryKey
    @Unique(onUniqueConflict = ConflictAction.IGNORE)
    public String id;

    @Column()
    public String name;

    @Column()
    @SerializedName("tag")
    public String category;

    @Column()
    public boolean active;
}`

so the idea is that if I try to insert a "tag" that is already in the database, the operation should be rejected... however the operation is always successful and the values of the row are always overwritten.

This is the operation that saves the new "tags" and that should ignore the ones already inserted:

`ProcessModelTransaction<Tag> transactionTags
                        = new SaveModelTransaction<>(
                        ProcessModelInfo.withModels(listTags));
                transactionTags.onExecute();`

Any idea of why all the rows already in the database are being updated?

Thanks!

**Comment by agrosner** [comment]
looks like a bug. If you share your `_Adapter`'s `getCreationQuery()` that would confirm it.

**Comment by Hannibalem** [comment]
Here is the method in the Adapter:

```
@Override
  public final String getCreationQuery() {
    return "CREATE TABLE IF NOT EXISTS `Tag`(`id` TEXT UNIQUE,`name` TEXT,`category` TEXT,`active` INTEGER, PRIMARY KEY(`id`)" + ");";
  }
```

**Comment by agrosner** [comment]
should be fixed in `3.0.0-beta6`. now `UNIQUE` clause includes an `ON CONFLICT` clause too.
